### PR TITLE
vis: Add types for Options.locales

### LINKED ISSUE #18687: @types/vis locales should not be string array
I am using last version of types/vis

The locales attribute of the Options interface should be "any" data type instead of string[]

locales?: string[];

vis.js won´t allow this type, only an object is allowed (http://visjs.org/docs/network/)

FYI
@andy-ms @sandersn

Using package.json to import VIS typings into the project

### PR DESCRIPTION
**Pull request by sandersn**
Fixes #18687 
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://visjs.org/docs/network/#locales

### CODE REVIEW

File: types/vis/index.d.ts [hunk 1]
```diff
@@ -1703,6 +1703,33 @@ export interface Edge {
...
+    addNode?: string;
+    addEdge?: string;
+    editNode?: string;
+    editEdge?: string;
+    addDescription?: string;
+    edgeDescription?: string;
+    editEdgeDescription?: string;
+    createEdgeError?: string;
+    deleteClusterError?: string;
+    editClusterError?: string;
```
Comment (macleodbroad-wf) [inline types/vis/index.d.ts]: Are these all really optional? I would expect that they are all required.

Comment (MichaelBitard) [inline types/vis/index.d.ts]: According to here: http://visjs.org/docs/network/index.html?#locales you're right, it's mandatory

**Comment by sandersn** [comment]
I fixed the failures and made the properties required.

**Comment by macleodbroad-wf** [comment]
+1

**Pull request merged**